app/client/packages/design-system/widgets/src/components/Icon/src/Icon.tsx

```
@@ -29,14 +32,14 @@ const _Icon = (props: IconProps, ref: Ref<Component>) => {
     <Suspense
       fallback={
         <FallbackIcon
-          className={styles.icon}
+          className={clsx(styles.icon, className)}
           data-size={Boolean(size) ? size : undefined}
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)